### PR TITLE
[client] fix export-file-stix list export (#connectors/1387)

### PR DESCRIPTION
**Pull request by SouadHadjiat**
### Proposed changes

**Related to :** https://github.com/OpenCTI-Platform/connectors/pull/1388

* remove unused parameters
* Fix export of relationships (we used to put standard_id in object_to_get instead of the entity object)

### Related issues

* https://github.com/OpenCTI-Platform/connectors/issues/1387

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality
- [ ] I wrote test cases for the relevant uses case
- [ ] I added/update the relevant documentation (either on github or on notion)
- [ ] Where necessary I refactored code to improve the overall quality

### Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by SarahBocognano — APPROVED** [review]

File: pycti/utils/opencti_stix2.py [hunk 1]
```diff
@@ -1609,23 +1609,21 @@ def prepare_export(
             entity["count"] = entity["attribute_count"]
             del entity["attribute_count"]
             entity["sighting_of_ref"] = entity["from"]["standard_id"]
-            objects_to_get.append(entity["from"]["standard_id"])
+            objects_to_get.append(entity["from"])
```
Comment (richard-julien) [inline pycti/utils/opencti_stix2.py:1612]: Really not sure about that. objects must contains identifiers i think

File: pycti/utils/opencti_stix2.py [hunk 2]
```diff
@@ -2225,9 +2220,6 @@ def export_selected(
             "id": "bundle--" + str(uuid.uuid4()),
             "objects": [],
         }
-        if entity_type == "StixFile":
```
Comment (richard-julien) [inline pycti/utils/opencti_stix2.py:2228]: If I remember well, this need to be kept

**Review by richard-julien — CHANGES_REQUESTED** [review]

File: pycti/utils/opencti_stix2.py (hunk 2 above)
Comment (RomuDeuxfois) [reply pycti/utils/opencti_stix2.py:2228]: I confirm
https://github.com/OpenCTI-Platform/client-python/issues/385

Comment (RomuDeuxfois) [reply pycti/utils/opencti_stix2.py:2228]: It's handle by generate_export method
So, it's ok ✅

**Review by RomuDeuxfois — APPROVED** [review]

File: pycti/utils/opencti_stix2.py (hunk 1 above)
Comment (RomuDeuxfois) [inline pycti/utils/opencti_stix2.py:1612]: Loop on and check complete object in line 1861 ✅

**Review by RomuDeuxfois — APPROVED** [review]

**Review by richard-julien — APPROVED** [review]

**Pull request merged**